packages/react-magma-dom/src/theme/magma.ts

```diff
@@ -291,6 +291,8 @@ export interface ThemeInterface {
   headingFont: string;
   iconSizes: IconSizes;
   iterableColors: string[];
+  chartColors?: string[];
+  chartInverseColors?: string[];
```

Comment:
Line 777 calls it `chartColorsInverse`. Don't care which name we use, but it has to be the same :)